draft-ietf-dnsop-svcb-https.md

```diff
@@ -659,6 +659,35 @@ each RRSet in the Additional section with the same DNSSEC-related records
 that they would send when providing that RRSet as an Answer (e.g. RRSIG, NSEC,
 NSEC3).
 
+## EDNS Client Subnet (ECS)
+
+The EDNS Client Subnet extension (ECS, {{!RFC7871}}) allows recursive
+resolvers to request IP addresses that are suitable for a particular client
+IP range.  IP addresses appear in two places in Authoritative SVCB responses:
+
+* ipv*hint SvcParams in a ServiceMode response.  Clients only use these hints
+when the recursive resolver does not provide A/AAAA records for TargetName in
+the Additional section, as recommended in {{recursive-behavior}}.
+* A/AAAA records returned by the Authoritative server in the Additional
+section of a SVCB response when TargetName is in-zone, as recommended in
+{{authoritative-behavior}}.
+
+SVCB records SHOULD NOT otherwise vary based on ECS.
+
+A recursive resolver that always returns A/AAAA records for TargetName in the
+Additional section, as recommended in {{recursive-behavior}}, SHOULD include
+an ECS extension with a SOURCE PREFIX-LENGTH of zero in SVCB queries, to
+indicate that ECS is supported but is disabled for this query.  If the
+resolver might not return these Additional records, it SHOULD send the
+same ECS extension that it would use for an A/AAAA query.
+
```

Comment:
Do we also want a bullet/entry suggesting that recursive resolvers implementing ECS for client performance MAY/SHOULD disable sending Additional A/AAAA records to clients in-favor of being able to use ECS to authorities?  The DNS performance may be better but the overall experience/performance could be much worse.

Reply:
I don't think that makes sense.  If the recursive resolver disables ECS for SVCB, clients will still be using ECS-derived A and AAAA records, so connection performance should not be impaired.

Reply:
I've reworked the text since this comment.  It now discusses this topic (line 679).